index.js

```diff
@@ -297,6 +297,14 @@ function prepareRequest(options) {
       case 'tempDir': {
         break;
       }
+      case 'overwrite': {
+        // the presence of any value for this flag causes the API to interpret it as
+        // true, so only add it to the payload if it is truthy
+        if (options.overwrite) {
+          formData[name] = String(value);
+        }
```

Comment:
If I make a curl call with `overwrite=false`, would the API interpret that value as truthy? If so, we should update our docs to make that clearer.

Reply:
yes